.github/workflows/monthly-builds.yaml

```diff
@@ -66,16 +66,17 @@ jobs:
           git init /home/runner/work/ros-content-sharing-snaps/ros-content-sharing-snaps/${{ matrix.snap_names }}
       - name: Print Launchpad build repository
         run: |
-          echo "Building at: https://git.launchpad.net/~ce-certification-qa/+snap/$SNAPCRAFT_BUILDER_ID"
+          echo "Building at: https://git.launchpad.net/~ce-certification-qa/+snap/${{ env.SNAPCRAFT_BUILDER_ID }}"
```

Comment:
what is this ~ce-certification-qa ?

Reply:
looks like a copy paste from the action of the certification team.
Even if we change it, the information/URL provided by snapcraft remote-build isn't valid. It looks like it's only valid for repository hosted on LP.
I am in favour of simply deleting it.